es/retroalimentacion.md

```diff
@@ -41,7 +41,7 @@ Para más información sobre los "issues", lee la guía de GitHub en [Mastering
 
 ## Opción 3: sugiere un cambio específico
 
-Para sugerir o hacer un cambio menor a una lección, primero navega al [directorio de lecciones](https://github.com/programminghistorian/jekyll/tree/gh-pages/lessons) y entra a la lección en cuestión.
+Para sugerir o hacer un cambio menor a una lección, primero navega al [directorio de lecciones](https://github.com/programminghistorian/jekyll/tree/gh-pages/en/lessons) y entra a la lección en cuestión.
```

Comment:
Why `en` here?

Reply:
Because we have moved English lessons into the `/en` directory, we have to explicitly change the links to corresponding paths in our github repository. If you to to <https://github.com/programminghistorian/jekyll/tree/gh-pages/lessons> right now, it's a 404.

We do want to be linking to the folder containing markdown for the English lessons here, right?

Reply:
I see. But why not https://github.com/programminghistorian/jekyll/tree/gh-pages/es/lecciones ?

Reply:
Sure we can swap it. I'd been maintaining the original translation content, but if you'd like to change it that is ok with me. Yes?